packages/one-bx-shell-app/src/assets/config/plugins.json

```diff
@@ -1,16 +1,63 @@
 [
   {
-    "name": "contentReq.testCobra",
+    "name": "contentReq",
     "provider": "Ariba",
     "version": "1.0.1",
-    "remoteEntry": "http://localhost:4201/remoteEntry.js",
-    "remoteName": "contentReqApp",
-    "exposedModule": "./Download",
-    "displayName": "DownloadComponent",
-    "componentName": "DownloadComponent",
-    "scope": "Page",
-    "category": "main",
-    "hasRoutes": false
+    "host": "http://localhost:4201",
+    "components": {
+      "prlist": {
+        "name": "PrListComponent",
+        "module": "./PrList",
+        "entry": "/remoteEntry.js",
+        "iframe": "/index.html",
+        "scope": "Page",
+        "category": "main",
+        "hasRoutes": false
+      },
+      "testCobra": {
```

Comment:
for Iframe there is nto name, no module, no entry there is only URL

Reply:
this will be updated as a part plugin schema task